src/front/wgsl/mod.rs

```diff
@@ -1047,9 +1041,7 @@ impl Composition {
 }
 
 #[derive(Default)]
-struct TypeAttributes {
-    stride: Option<NonZeroU32>,
-}
+struct TypeAttributes {}
```

Comment:
This deserves a comment:

> Although WGSL has no type attributes at the moment, it has had them in the past and may well acquire some again in the future, so just in case, we're leaving the plumbing in.